bciers/apps/reporting/src/middlewares/withRulesAppliedReport.ts

```diff
@@ -31,9 +56,52 @@ const handleIndustryUserRoutes = async (request: NextRequest, token: any) => {
       // 🛸 Redirect to BCIERS dashboard
       return NextResponse.redirect(new URL(`/onboarding`, request.url));
     }
+    const { pathname } = request.nextUrl;
+    const versionId = extractVersionId(pathname);
+
+    // Allow requests to "verification" if report needs verification
+    if (pathname.includes("verification")) {
```

Comment:
This might create a headache if we happen to have other routes that include the 'verification' down the line, and we'll start seeing random redirects. I suggest we are very explicit with the pathname pattern, maybe a regex match?